.gitignore

```diff
@@ -9,3 +9,7 @@ node_modules/*
 
 npm-debug.log
 yarn-error.log
+
+screenshots
+videos
+cypress.json
```

Comment:
I think this file should not be ingored

Reply:
I think screenshots and videos can be ignored or maybe we should discuss about this. And I have removed cypress.json.